maxim/logger/livekit/llm.py

```
@@ -0,0 +1,97 @@
+import functools
+import inspect
+import traceback
+
+from ...scribe import scribe
+from .store import get_session_store
+
+llm_f_skip_list = []
+
+
+def pre_hook(self, hook_name, args, kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `pre_hook`

Add return type annotation: `None`

(ANN201)